wallet/wallet.go

```diff
@@ -3759,7 +3759,12 @@ func isVote(tx *wire.MsgTx) bool {
 }
 
 func isRevocation(tx *wire.MsgTx) bool {
-	return stake.IsSSRtx(tx)
+	// isAutoRevocationsEnabled is set false to keep manually generated
+	// revocations still considered to be valid revocations.  Enabling this
+	// flag would cause legacy revocations to not be flagged as such.
+	// Enabling the flag only adds additional checks, so it's not necessary
+	// to make the call twice with the flag enabled and disabled.
```

Comment:
this comment applies to all of the changes throughout the wallet codebase as well where this flag is now necessary.